.gitignore

```diff
@@ -0,0 +1,2 @@
+**/yarn.lock
```

Comment:
I'm guessing you ran `yarn install` which is why this lock file is being generated. We aren't using yarn so generally stick to `npm install` and other npm commands

Reply:
Yeah I did 😂, so to prevent the mistake again.